packages/react/src/components/Checkbox/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+const Checkbox = ({
+  isUndefined,
+  disabled,
+  children,
+  isChecked: isDefaultChecked,
+  sublabel,
```

Comment:
What about `helper`, to match other form controls helpers?

Reply:
👍